content/yaml-quick-start/building-a-flutter-app.md

```diff
@@ -153,6 +149,15 @@ workflows:
         track: $GOOGLE_PLAY_TRACK
         in_app_update_priority: 0
 ```
+{{<notebox>}}
+Note: You can skip `Set up key properties` script if you are using [Code-signing Identities](https://docs.codemagic.io/yaml-code-signing/code-signing-identities/#android-keystores-1) and just add your keystore reference as follows in your yaml configuration.
```

Comment:
Maybe mention that the 'code signing identities' feature is available when using a team?

Reply:
Done